apps/angular/pipe-easy/src/app/app.component.ts

```diff
@@ -1,21 +1,21 @@
-import { NgFor } from '@angular/common';
+import { NgFor, TitleCasePipe } from '@angular/common';
 import { Component } from '@angular/core';
+import { ComputationPipe } from './computation.pipe';
 
 @Component({
   standalone: true,
-  imports: [NgFor],
+  imports: [NgFor, ComputationPipe, TitleCasePipe],
   selector: 'app-root',
   template: `
-    <div *ngFor="let person of persons; let index = index">
-      {{ heavyComputation(person, index) }}
-    </div>
+    @for (person of persons; track persons; let index = $index) {
+      <div>
+        {{ person | computation: index | titlecase }}
```

Comment:
you could now use a computed function 😅

Reply:
Should I convert my custom pipe to funtion ??? I did not understand a little bit.

Reply:
No no, it's just an alternative. but pipes are perfect here